bin/main.rb

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "Hello world!"
```

Comment:
- It is [recommended](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) to use `
 \#!/usr/bin/env ruby` to make the file executable, please :pray: before moving on, read through that article and implement that change